src/definitions/modules/checkbox.less

```
@@ -793,6 +793,62 @@
     }
 }
 
+& when (@variationCheckboxColored) {
+  /*--------------
+       Colored
+  ---------------*/
+
+  each(@colors, {
+    @color: replace(@key, '@', '');
+    @c: @colors[@@color][color];
+    @l: @colors[@@color][light];
+    @isVeryDark: @colors[@@color][isVeryDark];
+  
+    .ui.@{color}.checkbox input:checked ~ label::after {
+      color: @color;
+    }
+    .ui.@{color}.radio.checkbox input:checked ~ label::after {
+      background-color: @color;
+    }
+    .ui.@{color}.toggle.checkbox input:checked ~ label::before,
+    .ui.@{color}.slider.checkbox input:checked ~ label::before {
+      background-color: @color !important;
+    }
```

Beside the variable name: Any chance to increase specificity without using `!important`? (Haven't tested myself yet!)
```suggestion
    .ui.ui.@{color}.toggle.checkbox input:checked ~ label::before,
    .ui.ui.@{color}.slider.checkbox input:checked ~ label::before {
      background-color: @c;
    }
```

---

I tried `.ui.ui` and `.ui.ui.ui`, but still need that `!important` to work... 😞

---

It's because there is an existing `!important` at
```css
.ui.toggle.checkbox input:checked ~ label:before {
    background-color: #2185D0 !important;
}
```

We would need to remove the `!important` from there too

---

If you check the checkbox.less there are still a few `!important` for toggle and slider which are causing this. This PR is a good chance to get rid of them as well 😉